api.js

```diff
@@ -76,6 +77,13 @@ Api.prototype._handleTest = function (test) {
 	var isError = test.error.message;
 
 	if (isError) {
+		if (test.error.powerAssertContext) {
```

Comment:
Shouldn't this be handled more low-level, like in `test.js`? Would also be nice to put the logic in `enhanced-assert.js`.